app/utils/discovery.ts

```
@@ -0,0 +1,127 @@
+            },
+          }
+        );
+
+        let items: PlexItem[] = [];
+
+        if (sectionResponse.ok) {
+          const sectionContent = await sectionResponse.json();
+
+          // Extract items from the section
+          items =
+            sectionContent.MediaContainer?.Metadata?.map((item: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)